platform/android/export/export.cpp

```diff
@@ -2274,6 +2280,79 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 		return have_plugins_changed || first_build;
 	}
 
+	Error get_command_line_flags(const Ref<EditorExportPreset> &p_preset, const String &p_path, int p_flags, Vector<uint8_t> *command_line_flags) {
```

Comment:
```suggestion
	Error get_command_line_flags(const Ref<EditorExportPreset> &p_preset, const String &p_path, int p_flags, Vector<uint8_t> r_command_line_flags) {
```

Can't this be passed by reference and modified with its own API instead of a pointer being `copymem`'ed?